cli/src/plugins/javadoc/buildIndexes.ts

```diff
@@ -39,4 +39,25 @@ export const buildIndexes = async ({
     path: "/index",
     root,
   });
+
+  const packages = finalizedProject.entities
+    .filter((e) => e.data?.category === "package")
+
+  packages.forEach(pkg => {
+    const pkgRoot = md.root([
+      toctree(finalizedProject.entities.filter((e) => e.data?.category === "class").filter((e) => e.data?.containingPackage === pkg.canonicalName)
+        .map(child => {
+          const sanitizedName = child.canonicalName.replace(/#.*$/, "");
+          const value = sanitizedName.substring(sanitizedName.lastIndexOf('.') + 1) // only want simple class name
```

Comment:
This works, but there might be a way to access the original simplified name from the doc anyway. You can then add it to the user data on the entity.